editor.planx.uk/src/@planx/components/Notice/Public.tsx

```diff
@@ -75,6 +76,13 @@ const NoticeComponent: React.FC<Props> = (props) => {
     ? () => props.handleSubmit?.()
     : undefined;
 
+  const { trackResetFlow } = useAnalyticsTracking();
+
+  const handleNoticeResetClick = () => {
+    trackResetFlow()
+    props.resetPreview && props.resetPreview()
+  }
```

Comment:
nit: row end semi-colons in the body here would make this more readable for me? (i'm surprised linting/husky didn't pick this up actually!)